### DISCUSSION TITLE
Stop showing banner about new achievements if I have not actually got a new achivements

### DISCUSSION THREAD

**Discussion by matkoniecz** [community/community#19656] [Other Feature Feedback, Questions, & Ideas]
I am getting banner asking me to show achievements (I disabled them due to #18590), with claim that I got new achievements.

This claim turned out to be a lie after I enabled them temporarily.

I got this lying banner again:

![screen05](https://user-images.githubusercontent.com/899988/176307480-eb24846e-35b5-4581-a88b-c24905f2af57.png)

**Comment by FahdSeddik** [comment]
Hey!
It appears to pop up from time to time. Try clicking on the settings link and then close the banner. Hope this helps!
